app/Http/Controllers/Admin/UserContrller.php

```diff
@@ -0,0 +1,81 @@
+<?php
+namespace App\Http\Controllers\Admin;
+use App\Http\Controllers\Controller;
+use App\Http\Requests\Admin\UserRequest;
+use App\Http\Requests\SendMailUserProfileRequest;
+use Illuminate\Support\Facades\Session;
+use Illuminate\Mail\Mailable;
+use App\Services\MailService;
+use App\Mail\InformUserProfile;
+use App\Rules\Validate;
+use Mail;
+
+class UserContrller extends Controller
+{
+    public $listuser;
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $this->listuser = session()->get('users');
```

Comment:
Em dùng ntn nó không có ý nghĩ
Vì $this->listuser đang ko sử dụng lại ở đâu

Reply:
return view('admin.users.index', ['users' => $this->listuser]);

=>

return view('admin.users.index', ['users' => collect(session()->get('users'))])